### DISCUSSION TITLE
WxW

### DISCUSSION THREAD

**Discussion by giscus** [Whbbit1999/wxw-blog-comments#2] [General]
# WxW

关于我  人生幼时离家求学，每半月而还。青年求学县城，中等偏下。时临高考，名落孙山。技术学院，荒废三年。三月工厂生活，深入人间，感身薄体弱，叹人生之困苦。游学半年，找到程序工作，努力工作，认真学习，遂成前端组长。 叹人生之事，努力坚持遂能成之八九。在此勉励诸君，切勿轻言放弃。 近日喜欢的诗作我咽下一枚铁做的月亮许立志我咽下一枚铁做的月亮他们把它叫做螺丝我咽下这工业的废水，失业的订单那些低于机台的青

http://localhost:4000/about/

**Comment by Whbbit1999** [comment]
测试评论系统

**Comment by mytds** [comment]
博主好，最近也在使用 Stellar 主题，配置遇到了点问题，可以帮忙看下吗？ 有偿。

**Reply by Whbbit1999** [reply]
我可以帮你看看

**Reply by mytds** [reply]
100566876QQ和微信都是哈~

**Reply by Whbbit1999** [reply]
加你了，同意一下吧
